### PR TITLE
updated AppsUpdate view with correct GET request

### PR DESCRIPTION
**Pull request by reablaz**
So user can request invoice data by invoice id from btcpayserver, to verify callback

### CODE REVIEW

**Comment by reablaz** [comment]
@Kukks i am not familar with c#... also with co-op coding on git (it is like my one of the first experiences). so please let me know if i need to add myself something or you done edit already.

**Comment by Kukks** [comment]
It's fine, I will do it after then. :)

**Comment by reablaz** [comment]
Thank you, @Kukks  !

**Comment by NicolasDorier** [comment]
ping @Kukks can update this PR?

File: BTCPayServer/Views/Apps/UpdatePointOfSale.cshtml [hunk 1]
```diff
@@ -208,7 +208,7 @@
                                         <p><strong>Never</strong> trust anything but <code>id</code>, <strong>ignore</strong> the other fields completely, an attacker can spoof those, they are present only for backward compatibility reason:</p>
                                         <p>
                                             <ul>
-                                                <li>Send a <code>GET</code> request to <code>https://btcpay.example.com/invoices/{invoiceId}</code> with <code>Content-Type: application/json</code></li>
+                                                <li>Send a <code>GET</code> request to <code>https://btcpay.example.com/invoices/{invoiceId}</code> with <code>Content-Type: application/json; Authorization: Basic YourLegacyAPIkey"</code>, Legacy API key can be created with Access Tokens in Store settings</li>
```
Comment (NicolasDorier) [inline BTCPayServer/Views/Apps/UpdatePointOfSale.cshtml]: that's wrong, `YourLegacyAPIkey` should be base64 encoded.

**Comment by NicolasDorier** [comment]
I think you removed a commit here you only changed the comment @Kukks

**Pull request merged**